### DISCUSSION TITLE
Should the milestones section be used?

### DISCUSSION THREAD

**Discussion by ghost** [oxen-io/session-desktop#2037] [Ideas]
I think it benefit both sides of the developer/user coin, as in one side users can track the overall progress of an upcoming version and take a peek at what features, fixes and tweaks that may ne coming to the next update. On the other hand developers can grab issues and put it to whatever milestone tab (1.x.x) they want based on the difficulty of said task. What do ya'll think?

**Comment by peepo5** [comment]
:shrug:
